assets/js/callback.js

```diff
@@ -1 +1,37 @@
-alert("authentication callback complete.")
+// alert("authentication callback complete.")
+
+function redirectUser() {
+
+  let redirectUrl
+
+  const params = new URLSearchParams(window.location.search)
+  const referringUrl = decodeURIComponent(params.get("ref"))
+  // const hasProject = decodeURIComponent(params.get("projects"))
+  const userToken = decodeURIComponent(params.get("access-token"))
```

Comment:
The token may be in `idTokenPayload` in localStorage or on the window object, depending on how you've configured your own login callback. Make sure you have access to what you are looking for.

Reply:
There is access token in the window object and usertoken in the localstorage. I no longer have to pass token via url. I have refactored this.
However, it is unclear which of the two tokens will be required to make the call. I think either would serve though, especially access token